### PR TITLE
Fix command is all comment

### PR DESCRIPTION
**Pull request by laura-ding**
as title

### CODE REVIEW

**Comment by laura-ding** [comment]
jenkins go

File: src/parser/scanner.lex [hunk 1]
```diff
@@ -310,7 +310,11 @@ IP_OCTET                    ([0-9]|[1-9][0-9]|1[0-9][0-9]|2[0-4][0-9]|25[0-5])
 "--".*                      // Skip the annotation
 "/*"                        { BEGIN(COMMENT); }
 <COMMENT>"*/"               { BEGIN(INITIAL); }
-<COMMENT>([^*]|\n)+|.       // Skip the annotation
+<COMMENT>([^*]|\n)+|.
+<COMMENT><<EOF>>            {
+                                // Must match /* */
+                                throw GraphParser::syntax_error(*yylloc, "incomplete comment");
```
Comment (dutor) [inline src/parser/scanner.lex]: Better change the error to _unterminated comment_

Comment (dutor) [reply src/parser/scanner.lex]: Oh, by the way, could you please handle the unterminated case for `STRING`?

File: src/common/base/Status.h [hunk 2]
```diff
@@ -121,7 +123,8 @@ class Status final {
         kNoSuchFile             = 102,
         kNotSupported           = 103,
         // 2xx, for graph engine errors
-        kSyntaxError            = 301,
+        kSyntaxError            = 201,
+        kSyntaxDoNone           = 202,
```
Comment (dutor) [inline src/common/base/Status.h]: Maybe `kStatementEmpty`?

File: src/parser/scanner.lex [hunk 3]
```diff
@@ -280,6 +280,10 @@ IP_OCTET                    ([0-9]|[1-9][0-9]|1[0-9][0-9]|2[0-4][0-9]|25[0-5])
                                 }
                                 return TokenType::STRING;
                             }
+<DQ_STR><<EOF>>             {
+                                // Must match ""
+                                throw GraphParser::syntax_error(*yylloc, "unterminated string \"\"");
```
Comment (dutor) [inline src/parser/scanner.lex]: `"unterminated string"` is enough.

File: src/parser/scanner.lex [hunk 4]
```diff
@@ -288,6 +292,10 @@ IP_OCTET                    ([0-9]|[1-9][0-9]|1[0-9][0-9]|2[0-4][0-9]|25[0-5])
                                 }
                                 return TokenType::STRING;
                             }
+<SQ_STR><<EOF>>             {
+                                // Must match ''
+                                throw GraphParser::syntax_error(*yylloc, "unterminated string ''");
```
Comment (dutor) [inline src/parser/scanner.lex]: ditto.

Comment (dutor) [reply src/parser/scanner.lex]: You could have combined these two _unterminated_ checking in one.

File: src/graph/test/SchemaTest.cpp [hunk 5]
```diff
@@ -27,6 +27,24 @@ class SchemaTest : public TestBase {
...
+    // Test command is comment
+    {
+        cpp2::ExecutionResponse resp;
+        std::string cmd = "# CREATE TAG TAG1";
+        auto code = client->execute(cmd, resp);
+        ASSERT_EQ(cpp2::ErrorCode::E_STATEMENT_EMTPY, code);
+    }
+    {
+        cpp2::ExecutionResponse resp;
+        std::string cmd = "SHOW SPACES # show all spaces";
```
Comment (whitewum) [inline src/graph/test/SchemaTest.cpp]: BTW, add the case :
"-- SHOW SPACES; SHOW SPACES;"
E_STATEMENT_EMTPY

File: src/interface/graph.thrift [hunk 6]
```diff
@@ -25,6 +25,8 @@ enum ErrorCode {
 
     E_SYNTAX_ERROR = -7,
     E_EXECUTION_ERROR = -8,
+    // Nothing is executed When command is comment
```
Comment (whitewum) [inline src/interface/graph.thrift]: what is the comment style?  // should the comment be placed after the error code in the same line?

File: src/graph/test/SchemaTest.cpp (hunk 5 above)
Comment (laura-ding) [reply src/graph/test/SchemaTest.cpp]: It's the same as #

**Pull request merged**